apps/kitchen-sink/src/ensemble/screens/widgets.yaml

```diff
@@ -20,6 +20,29 @@ View:
             text: Widgets
         - Markdown:
             text: More to come! In the meantime, checkout the Ensemble [documentation](https://docs.ensembleui.com/).
+
+        - Markdown:
+            text: "## Collapse"
+        - Collapsible:
+            id: collapseWidget
+            value:
+              - "1"
+            expandIconPosition: "end"
```

Comment:
How about supporting either a header widget and allow them to customize the header entirely OR support a title and icon

Reply:
I already added a support for that, user can pass any widget as a header for each collapsible item.
for example,
```js
items:
  - key: "1"
    label:
      Text:
        text: Here is my label 1
    children: hiii
  - key: "2"
    label: Here is my label 2
    children:
      Text:
        text: Here is my value 2
```

here label represent the header of collapsible and children represent the content which will be shown when the item will be collapse